shell/common/shell.cc

```diff
@@ -488,7 +489,8 @@ std::unique_ptr<Shell> Shell::Spawn(
         PlatformData{}, task_runners_, rasterizer_->GetRasterThreadMerger(),
         GetSettings(), vm_, vm_->GetVMData()->GetIsolateSnapshot(),
         on_create_platform_view, on_create_rasterizer,
-        [engine = this->engine_.get()](
+        [engine = this->engine_.get(),
+         initial_route = std::move(initial_route)](
```

Comment:
Same here for std::move.  Since this is executing on a separate thread, just explicitly make a copy.

Reply:
Done